projects/memapCore/src/main/java/memap/helper/EnergyPrices.java

```diff
@@ -61,34 +70,23 @@ public static EnergyPrices getInstance() {
 	 * 
 	 * @param csvFile
 	 */
-	private void setEnergyPrices(String csvFile) {
+	private void setElectricityPrices(String csvFile) {
 		try {
 			if (csvFile.isEmpty()) {
-				readElectricityPrices(getBuffer("ELECTRICITYPRICEEXAMPLE"));
+				readEnergyPrices(electricityPrices, getBuffer("ELECTRICITYPRICEEXAMPLE"));
 				System.err
 						.println("Variable market price selected but not input file was provided. Using example file");
 			} else {
-				readElectricityPrices(getBuffer(csvFile));
```

Comment:
This is merely a cosmetic comment, but up to now the "readElectricityPrices()"/"readEnergyPrices()" method is only used for electricity. So we can keep the elecricity as part of the name, or do you prefer to keep is more general?

Reply:
I would suggest to keep it general, because this method is only interpolating the input ArrayList independent of commodity.
You could rather call it InterpolatePrices() or similar....

Reply:
allright, then lets keep it that way.